src/gui/advancedsettings.cpp

```diff
@@ -376,7 +379,13 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
     addRow(ASYNC_IO_THREADS, (tr("Asynchronous I/O threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#aio_threads", "(?)"))
             , &m_spinBoxAsyncIOThreads);
-
+    // stop tracker timeout
+    m_spinBoxStopTrackerTimeout.setMinimum(1);
```

Comment:
`0` is a valid value for the setting and the default minimum is also `0` so you should remove this line.